sydent/db/invite_tokens.py

```diff
@@ -53,7 +83,70 @@ def getTokens(self, medium, address):
 
         return ret
 
+    def getTokensAfterId(self, afterId, limit):
+        """Retrieves max `limit` invite tokens after a given DB id.
+        
+        :param afterId: A database id to act as an offset. Tokens after this id are returned.
+        :type afterId: int
+        :param limit: Max amount of database rows to return.
+        :type limit: int
+        :returns a tuple consisting of a list of invite tokens and the maximum DB id that was extracted.
```

Comment:
also: can you add an `:rtype`? (I think the type is `Tuple[dict[int, str], int|None]`)

Reply:
`maxId` is 0 if no rows were found (perhaps we should change this). But otherwise looks correct.

Reply:
Ah, your next comment suggest to change it to None. I agree.